### ISSUE TITLE
check that all methods in a test are annotated

### ISSUE THREAD

**Issue by yegor256** [volodya-lombrozo/jtcop#294]
Would be great to check whether all methods in a test are annotated with `@Test` or `@Before` or `@After`

**Comment by yegor256** [comment]
@volodya-lombrozo it was missing at https://github.com/yegor256/jhome/commit/f27131ce060260557a1c8530ad1febf590c38ec2#diff-4ae68b20587162945777266357143ea2da59b1cb31b71e5f40cc5db2325f7d86R114 That's why I suggested to add it. If it already exists, let's close this ticket.

**Comment by volodya-lombrozo** [comment]
@yegor256 you are right. The check doesn't work by some reason. I will investigate it and fix. Thanks.
